internal/controller/payload_handler.go

```diff
@@ -72,7 +72,8 @@ func (ph PayloadHandler) HandleMessage(ctx context.Context, m protocol.Message)
 		return nil
 	}
 
-	ph.Writer.WriteMessages(ctx,
+	kw := queue.StartProducer(queue.GetProducer())
```

Comment:
Do we want to create a new Kafka producer each time we handle a message?

Reply:
Just saw the commit message. Assuming this is temporary then

Reply:
Yep.  I temporarily crammed/hardcoded it in there.  :(